bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -439,20 +302,30 @@ All these channels are read only.
 
 | Channel ID          | Item Type          | Description                                                |
 |---------------------|--------------------|------------------------------------------------------------|
-| Temperature         | Number:Temperature | Current temperature                                        |
-| Sp_Temperature      | Number:Temperature | Thermostat temperature setpoint                            |
-| SetpointMode        | String             | Chosen setpoint_mode (program, away, hg, manual, off, max) |
-| Planning            | String             | Id of the currently active planning when mode = program    |
+| temperature         | Number:Temperature | Current temperature                                        |
+| setpoint            | Number:Temperature | Thermostat temperature setpoint                            |
+| setpoint-mode       | String             | Chosen setpoint mode (program, away, hg, manual, off, max) |
 | ThermRelayCmd       | Switch             | Indicates whether the furnace is heating or not            |
-| ThermOrientation    | Number             | Physical orientation of the thermostat module              |
-| TimeStamp           | DateTime           | Timestamp when data was measured                           |
-| SetpointEndTime     | DateTime           | Thermostat goes back to schedule after that timestamp      |
-| LastMessage         | DateTime           | Last message emitted by the module                         |
-| LowBattery          | Switch             | Low battery                                                |
-| BatteryVP           | Number             | Battery level                                              |
-| RfStatus            | Number             | Signal strength (0 for no signal, 1 for weak, 2 for average, 3 for good or 4 for excellent) |
+| anticipating        | Switch             | Indicates is anticipating the schedule                     |
+| timestamp           | DateTime           | Timestamp when data was measured                           |
+| setpoint-end        | DateTime           | Thermostat goes back to schedule after that timestamp      |
+| last-message        | DateTime           | Last message emitted by the module                         |
+| signal              | strength           | Number               | Signal strength (0 for no signal, 1 for weak...) |
+| signal              | value              | Number:Power         | Signal strength in dBm                           |
+| battery             | value              | Number               | Battery level                                    |
+| battery             | low-battery        | Switch               | Low battery                                      |
+| battery             | status             | String               | Description of the battery status (*)            |
```

Comment:
Something is wrong here, it seems that first and second columns should have been only first column.

Reply:
Corrected